src/main/webapp/sass/_core.scss

```diff
@@ -227,7 +227,8 @@ a:focus {
       }
       button.toolsMenuButton,
       .dropdown.open,
-      .MuiButton-root {
+      .MuiButton-root,
+      .MuiIconButton-root {
```

Comment:
Also include MuiIconButtons (in addition to MuiButtons) when styling buttons under the projectMetadata class.  This makes the icon color white